resources/xlf/en/mssql.xlf

```diff
@@ -359,35 +368,290 @@
       <source xml:lang="en">Indicates which server type system then provider will expose through the DataReader</source>
     </trans-unit>
   </body></file>
-  <file original="extensions/mssql/out/sparkFeature/dialog/sparkJobSubmission/sparkJobSubmissionService" source-language="en" datatype="plaintext"><body>
+  <file original="extensions/mssql/dist/localizedConstants" source-language="en" datatype="plaintext"><body>
+    <trans-unit id="msgMissingNodeContext">
+      <source xml:lang="en">Node Command called without any node passed</source>
+    </trans-unit>
+    <trans-unit id="mssql.manageAccessTitle">
+      <source xml:lang="en">Manage Access</source>
+    </trans-unit>
+    <trans-unit id="mssql.permissionsTitle">
+      <source xml:lang="en">Permissions</source>
+    </trans-unit>
+    <trans-unit id="mssql.ownersHeader">
+      <source xml:lang="en">Owners</source>
+    </trans-unit>
+    <trans-unit id="mssql.allOthersHeader">
+      <source xml:lang="en">All Others</source>
+    </trans-unit>
+    <trans-unit id="mssql.everyone">
+      <source xml:lang="en">Everyone</source>
+    </trans-unit>
+    <trans-unit id="mssql.userLabel">
+      <source xml:lang="en">User</source>
+    </trans-unit>
+    <trans-unit id="mssql.groupLabel">
+      <source xml:lang="en">Group</source>
+    </trans-unit>
+    <trans-unit id="mssql.accessHeader">
+      <source xml:lang="en">Access</source>
+    </trans-unit>
+    <trans-unit id="mssql.defaultHeader">
+      <source xml:lang="en">Default</source>
+    </trans-unit>
+    <trans-unit id="mssql.stickyHeader">
+      <source xml:lang="en">Sticky</source>
+    </trans-unit>
+    <trans-unit id="mssql.readHeader">
+      <source xml:lang="en">Read</source>
+    </trans-unit>
+    <trans-unit id="mssql.writeHeader">
+      <source xml:lang="en">Write</source>
+    </trans-unit>
+    <trans-unit id="mssql.executeHeader">
+      <source xml:lang="en">Execute</source>
+    </trans-unit>
+    <trans-unit id="mssql.addUserOrGroup">
+      <source xml:lang="en">Add User or Group</source>
+    </trans-unit>
+    <trans-unit id="mssql.enterNamePlaceholder">
+      <source xml:lang="en">Enter name</source>
+    </trans-unit>
+    <trans-unit id="mssql.addLabel">
+      <source xml:lang="en">Add</source>
+    </trans-unit>
+    <trans-unit id="mssql.namedUsersAndGroups">
+      <source xml:lang="en">Named Users and Groups</source>
+    </trans-unit>
+    <trans-unit id="mssql.apply">
+      <source xml:lang="en">Apply</source>
+    </trans-unit>
+    <trans-unit id="mssql.applyRecursively">
+      <source xml:lang="en">Apply Recursively</source>
+    </trans-unit>
+    <trans-unit id="mssql.errorApplyingAclChanges">
+      <source xml:lang="en">Unexpected error occurred while applying changes : {0}</source>
+    </trans-unit>
+    <trans-unit id="sparkJobSubmission_LocalFileDestinationHint">
```

Comment:
The other on is this issue (sorry for not commenting earlier) this will cause the actual lang xlf s to contain same keys since it drop anything after _

Reply:
As discussed this is status quo